bin/deploy-local

```diff
@@ -0,0 +1 @@
+helm upgrade mine-support --install --namespace mine-support --values ./helm/dev-values.yaml ./helm --set auth=$MINE_BASIC_AUTH
```

Comment:
This need a shebang at the top, and the file wasn't chmoded as executable when I fetched it

#!/bin/sh

Reply:
thanks, I'll take a look

Reply:
fixed